int32 object_label
sensor_msgs/PointCloud[] segments
geometry_msgs/Pose[] 	   ego_poses
